src/backend/src/utils/auth.utils.ts

```
@@ -31,6 +31,18 @@ export const requireJwtProd = (req: Request, res: Response, next: NextFunction)
     req.path === '/' || // base route is available so aws can listen and check the health
     req.method === 'OPTIONS' // this is a pre-flight request and those don't send cookies
   ) {
+    next();
+  } else if (
+    req.path === '/tasks/sendTaskDeadlineSlackNotifications' // task deadline notification endpoint
+  ) {
+    const { authorization } = req.headers;
+    const { NOTIFICATION_ENDPOINT_SECRET } = process.env;
```

you should check if this is defined and throw an error if it's undefined